orttraining/orttraining/core/graph/zero_optimizer_graph_builder.cc

```diff
@@ -275,7 +276,7 @@ static Status AddParameterPartition(
 
     // Partition initial optimizer state
     if (enabled[i]) {
-      weight_partition_info[weight_argdef.name].view_name = weight_views[i].name;
+      weight_partition_info[weight_argdef.name].partition_name = weight_views[i].name;
```

Comment:
my understanding is, if we do zero partition for those megatron partitioned weights, then partition_name will have the megatron partitioned weight view name? Will that be okay for our ckpt aggregation?

Reply:
We use this partition_info field to get the name of the initializer that should be fetched only within ORT training session. The original weight names are used to save and aggregate the checkpoints, and the checkpoint aggregation does not rely on how we name the initializers internally.